src/com/example/ui.cljs

```diff
@@ -131,29 +131,66 @@
                  (p :.max-w-prose.mb-5.px-4
                     "Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.")))))
 
+(defn is-today? [date-str]
+  (let [today (js/Date.)
+        date (js/Date. date-str)]
+    (and (= (.getDate today) (.getDate date))
+         (= (.getMonth today) (.getMonth date))
+         (= (.getFullYear today) (.getFullYear date)))))
+
+(defsc Dashboard [this {:keys [readiness-quiz current-user] :root/keys [response]}]
+  {:route-segment ["dashboard"]
+   :query [{:readiness-quiz [:quiz/id]}
+           {[:root/response '_] [:response/total]}
+           {:current-user [{:responses [:response/id :response/total :response/created-at]}]}]
+   :ident (fn [] [:component/id :dashboard])
+   :initial-state {}
+   :will-enter (fn [app _]
+                 (dr/route-deferred [:component/id :dashboard]
+                                    #(df/load! app
+                                               [:component/id :dashboard]
+                                               Dashboard
+                                               {:post-mutation `dr/target-ready
+                                                :post-mutation-params {:target [:component/id :dashboard]}})))}
+  (let [todays-response (or response (first (filter (fn [r]
+                                                      (is-today? (:response/created-at r))) (:responses current-user))))
+        total (:response/total todays-response)]
```

Comment:
we either take the response they just recorded that is in our database or look for todays response in the ones we loaded and if we find one, we use that for "todays total"